frontend/app/src/pages/defi/airdrops/index.vue

```diff
@@ -35,8 +35,15 @@ type AirdropSources = {
   readonly [source in AirdropType]: AirdropSource;
 };
 
+const { t } = useI18n();
+
 const expanded: Ref<Airdrop[]> = ref([]);
 const selectedAccounts = ref<GeneralAccount[]>([]);
+const statusFilters = ref<{ text: string; value: boolean }[]>([
```

Comment:
Use `const statusFilters :Ref<{ text: string; value: boolean }[]> = ref([...])` as it's more efficient